packages/client/src/hooks/useTreasury.js

```diff
@@ -279,6 +279,16 @@ export default function useTreasury(treasuryAddr, userAddr) {
     await refreshTreasury();
   };
 
+  const renameTreasury = (treasuryAddr, newName) => {
+    dispatch({
+      type: "RENAME_TREASURY",
+      payload: {
+        address: treasuryAddr,
+        name: newName,
+      },
+    });
```

Comment:
I made `SET_TREASURY` a bit generic on purpose so you could be able to just call

```
dispatch({
      type: "SET_TREASURY",
      payload: {
        [address]: {
          name: newName,
        }
    });
```

Reply:
ah makes sense, removed new action and used this instead